Jenkinsfile

```diff
@@ -147,18 +126,22 @@ pipeline{
 				}
 			}
 		}
-		// This stage emails the contents of the GraphQA_Summary_vXX.csv that is generated by graph-qa for review. 
+		// This stage emails the contents of the GraphQA_Summary_vXX.csv that is generated by graph-qa for review, as well as the GraphQA_Summary_vXX-1 (previous release). 
 		stage('Post: Email graph-qa output'){
 			steps{
 				script{
+					def prevGraphQAFileName = "GraphQA_Summary_v${previousRelease}.csv"
+					def s3PathPrevGraphQA = "${env.S3_RELEASE_DIRECTORY_URL}/${previousRelease}/orthoinference/reports/${prevGraphQAFileName}.gz"
+					sh "aws s3 cp ${s3PathPrevGraphQA} ."
+					sh "gunzip ${prevGraphQAFileName}.gz"
 					emailext (
-						body: "Hello,\n\nThis is an automated message from Jenkins regarding an update for v${currentRelease}. The Orthoinference step has finished running. Attached to this email should be the summary report output by graph-qa. Please compare this with the graph-qa output from the previous release and confirm if they look appropriate with the developer running Release. \n\nThanks!",
+						body: "Hello,\n\nThis is an automated message from Jenkins regarding an update for v${currentRelease}. The Orthoinference step has finished running. Attached to this email should be the summary report output by graph-qa for both v${currentRelease} and v${previousRelease}. Please compare these and confirm if they look appropriate with the developer running Release. \n\nThanks!",
 						to: '$DEFAULT_RECIPIENTS',
 						from: "${env.JENKINS_RELEASE_EMAIL}",
 						subject: "Orthoinference graph-qa for v${currentRelease}",
-						attachmentsPattern: "**/graph-qa/reports/GraphQA_Summary_v${currentRelease}.csv"
-						
+						attachmentsPattern: "**/graph-qa/reports/GraphQA_Summary_v${currentRelease}.csv, **/${prevGraphQAFileName}"
```

Comment:
Could the path for the currentGraphQAFileName also get a variable?  I think the symmetry of both having a variable name would help the readability.